.github/workflows/data-update.yml

```diff
@@ -20,4 +21,5 @@ jobs:
         with:
           github-token: ${{ secrets.GH_ACCESS_TOKEN }}
           push-branch: main
+          files: reports/.
```

Comment:
Hu, would have assumed it did this by default. The docs were not clear on that behavior.

Reply:
Yeah, that was my guess too. It did not.